### PR TITLE
Simplify, combine components and remove unused

### PR DESCRIPTION
**Pull request by smyzin**
- combined two components in one
- simplified components
- remove unused components

### CODE REVIEW

**Review by backmeupplz — APPROVED** [review]

File: src/components/SigningFlow.tsx [hunk 1]
```diff
@@ -93,7 +92,9 @@ export default function () {
   const { title, subTitle } = SigningStates[flowState]
 
   return error ? (
-    <ErrorBlock
+    <StatusBlock
+      error
+      loadingText=""
```
Comment (T-Damer) [inline src/components/SigningFlow.tsx]: How about making `loadingText` an optional argument?

Comment (T-Damer) [reply src/components/SigningFlow.tsx]: Oh it's optional already, simply remove this

File: src/components/StatusBlock/StatusLoading.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}) {
+  if (!loadingText) return null
+
+  return (
+    <AccentText>
+      <StatusText color="success">
+        <span className={loadingBlock}>
+          {loadingText} {completed ? <Checkmark /> : <Spinner />}
+        </span>
+      </StatusText>{' '}
```
Comment (T-Damer) [inline src/components/StatusBlock/StatusLoading.tsx:27]: We don't need this space

File: src/components/StatusBlock/StatusLoading.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  loadingText?: string
+  completed?: boolean
+}) {
+  if (!loadingText) return null
+
+  return (
+    <AccentText>
+      <StatusText color="success">
+        <span className={loadingBlock}>
+          {loadingText} {completed ? <Checkmark /> : <Spinner />}
```
Comment (T-Damer) [inline src/components/StatusBlock/StatusLoading.tsx:25]: Can we move `{completed ? <Checkmark /> : <Spinner />}` to the next line?

**Review by T-Damer — APPROVED** [review]

File: src/components/SigningFlow.tsx (hunk 1 above)
Comment (smyzin) [reply src/components/SigningFlow.tsx]: Already did this 🙂

File: src/components/StatusBlock/StatusLoading.tsx (hunk 3 above)
Comment (smyzin) [reply src/components/StatusBlock/StatusLoading.tsx:25]: Yep

File: src/components/StatusBlock/StatusLoading.tsx (hunk 2 above)
Comment (smyzin) [reply src/components/StatusBlock/StatusLoading.tsx:27]: You're right

**Pull request merged**